### ISSUE TITLE
Maven failure

### ISSUE THREAD

**Issue by yossigil** [TechnionYP5777/SmartCity-Accessibility#40]
I get these two maven errors in eclipse:
 
- Description	Resource	Path	Location	Type
Missing artifact com.teamdev.jxmaps:jxmaps-license:jar:1.2	pom.xml	/SmartCity-Accessibility	line 62	Maven Dependency Problem

- Description	Resource	Path	Location	Type
Missing artifact com.teamdev.jxmaps:jxmaps:jar:1.2	pom.xml	/SmartCity-Accessibility	line 55	Maven Dependency Problem

running 'mvn build` on the command line gives this:
```java
[INFO] Scanning for projects...
[WARNING] 
[WARNING] Some problems were encountered while building the effective model for SmartCity-Accessibility:SmartCity-Accessibility:jar:0.0.1-SNAPSHOT
[WARNING] 'dependencies.dependency.systemPath' for com.teamdev.jxmaps:jxmaps:jar should not point at files within the project directory, ${project.basedir}/src/main/resources/jxmaps-1.2.jar will be unresolvable by dependent projects @ line 32, column 16
[WARNING] 'dependencies.dependency.systemPath' for com.teamdev.jxmaps:jxmaps:jar should not point at files within the project directory, ${project.basedir}/src/main/resources/jxmaps-linux32-1.2.jar will be unresolvable by dependent projects @ line 39, column 16
[WARNING] 'dependencies.dependency.(groupId:artifactId:type:classifier)' must be unique: com.teamdev.jxmaps:jxmaps:jar -> duplicate declaration of version 1.2 @ line 34, column 15
[WARNING] 'dependencies.dependency.systemPath' for com.teamdev.jxmaps:jxmaps:jar should not point at files within the project directory, ${project.basedir}/src/main/resources/jxmaps-linux64-1.2.jar will be unresolvable by dependent projects @ line 46, column 16
[WARNING] 'dependencies.dependency.(groupId:artifactId:type:classifier)' must be unique: com.teamdev.jxmaps:jxmaps:jar -> duplicate declaration of version 1.2 @ line 41, column 15
[WARNING] 'dependencies.dependency.systemPath' for com.teamdev.jxmaps:jxmaps:jar should not point at files within the project directory, ${project.basedir}/src/main/resources/jxmaps-mac-1.2.jar will be unresolvable by dependent projects @ line 53, column 16
[WARNING] 'dependencies.dependency.(groupId:artifactId:type:classifier)' must be unique: com.teamdev.jxmaps:jxmaps:jar -> duplicate declaration of version 1.2 @ line 48, column 15
[WARNING] 'dependencies.dependency.systemPath' for com.teamdev.jxmaps:jxmaps:jar should not point at files within the project directory, ${project.basedir}/src/main/resources/jxmaps-win-1.2.jar will be unresolvable by dependent projects @ line 60, column 16
[WARNING] 'dependencies.dependency.(groupId:artifactId:type:classifier)' must be unique: com.teamdev.jxmaps:jxmaps:jar -> duplicate declaration of version 1.2 @ line 55, column 15
[WARNING] 'dependencies.dependency.systemPath' for com.teamdev.jxmaps:jxmaps-license:jar should not point at files within the project directory, ${project.basedir}/src/main/resources/license.jar will be unresolvable by dependent projects @ line 67, column 16
[WARNING] 'dependencies.dependency.systemPath' for com.teamdev.jxmaps:jxmaps:jar refers to a non-existing file /home/yogi/git/SmartCity-Accessibility/src/main/resources/jxmaps-win-1.2.jar @ line 60, column 16
[WARNING] 'dependencies.dependency.systemPath' for com.teamdev.jxmaps:jxmaps-license:jar refers to a non-existing file /home/yogi/git/SmartCity-Accessibility/src/main/resources/license.jar @ line 67, column 16
[WARNING] 
[WARNING] It is highly recommended to fix these problems because they threaten the stability of your build.
[WARNING] 
[WARNING] For this reason, future Maven versions might no longer support building such malformed projects.
[WARNING] 
[INFO]                                                                         
[INFO] ------------------------------------------------------------------------
[INFO] Building SmartCity-Accessibility 0.0.1-SNAPSHOT
[INFO] ------------------------------------------------------------------------
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 0.440 s
[INFO] Finished at: 2016-12-09T09:26:28+02:00
[INFO] Final Memory: 9M/147M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal on project SmartCity-Accessibility: Could not resolve dependencies for project SmartCity-Accessibility:SmartCity-Accessibility:jar:0.0.1-SNAPSHOT: The following artifacts could not be resolved: com.teamdev.jxmaps:jxmaps:jar:1.2, com.teamdev.jxmaps:jxmaps-license:jar:1.2: Could not find artifact com.teamdev.jxmaps:jxmaps:jar:1.2 at specified path /home/yogi/git/SmartCity-Accessibility/src/main/resources/jxmaps-win-1.2.jar -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/DependencyResolutionException
```

**Comment by kapalex** [comment]
Please note [this wiki](https://github.com/TechnionYP5777/SmartCity-Accessibility/wiki/JxMaps-dependency).
As we are heavily dependent on the JxMaps library we've added it as a maven dependency.  It isn't available to be automatically downloaded since it's generally a paid software, so the files need to be added manually.
I can add your preferred gmail to the  shared folder on google drive so you can download the license.

**Comment by kapalex** [comment]
Closing it for now as it is not a bug, reopen if necessary

**Comment by yossigil** [comment]
Good point...
Perhaps it would have been cleared if this page was made part of README.MD? What do you reckon?

**Comment by kapalex** [comment]
Definitely true, adding as a bullet point to issue #41
